langchain/chains/retrieval_qa/base.py

```diff
@@ -108,7 +108,7 @@ def _call(self, inputs: Dict[str, str]) -> Dict[str, Any]:
 
         docs = self._get_docs(question)
         answer = self.combine_documents_chain.run(
-            input_documents=docs, question=question
+            input_documents=docs, **inputs
```

Comment:
hmmm the default self.input_key is `query`, i think that will break when combine_documents is question?

Reply:
So how about change default `self.input_key` to `question`? Or just add `question` to `inputs` dict?

Reply:
How about return `[self.input_key] + self.combine_documents_chain.input_keys` in `RetrievalQA.input_keys`. This will lead pre-check all input_keys  when doing querying(`query` and `question` by default). I think the original code that transforms the `input_key` into `question` a bit perplexing.